app/components/Blockchain/Transaction/Transaction.jsx

```diff
@@ -34,17 +39,45 @@ export default class Transaction extends React.Component<Props> {
         </div>
         <div className={styles.txLabelContainer}>{label}</div>
         <div className={styles.txAmountContainer}>{amount}</div>
-        <div className={styles.txToContainer}>{to}</div>
-        <span
-          className={classNames(styles.txidLink, className)}
+        <div className={styles.txToContainer}>
+          {to}
+          {to !== 'NETWORK FEES' && (
```

Comment:
both this and to === 'NETWORK FEES' can be combined in one ternary expression. to !== 'NETWORK FEES' ? () : ()

also, you might want to make NETWORK FEES a variable in case you change the text, it will apply to where you created it originally and inside this ternary as well

Reply:
Because the closing </div> is not in the ternary im going to leave these as two separate logical operators for now... but I did go ahead and leverage a boolean value passed down from the action instead of checking for equality against that hard coded string